code/game/machinery/status_display.dm

```diff
@@ -32,6 +36,55 @@
 	var/index1 // display index for scrolling messages or 0 if non-scrolling
 	var/index2
 
+/obj/item/wallframe/status_display
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon_state = "unanchoredstatusdisplay"
+	custom_materials = list(/datum/material/iron=14000, /datum/material/glass=8000)
+	result_path = /obj/machinery/status_display
+
+//Copy pasted decon paths from newscaster.dm
+/obj/machinery/status_display/attackby(obj/item/I, mob/living/user, params)
+	if(I.tool_behaviour == TOOL_WRENCH)
+		to_chat(user, span_notice("You start [anchored ? "un" : ""]securing [name]..."))
+		I.play_tool_sound(src)
+		if(I.use_tool(src, user, 60))
+			playsound(loc, 'sound/items/deconstruct.ogg', 50, TRUE)
+			if(machine_stat & BROKEN)
+				to_chat(user, span_warning("The broken remains of [src] fall on the ground."))
+				new /obj/item/stack/sheet/iron(loc, 5)
```

Comment:
Why does this give more than the other `deconstruct` place? Either way, this should not be here and will be handled by the `deconstruct()`